components/atom/upload/src/index.js

```diff
@@ -22,6 +25,10 @@ const AtomUpload = ({
   status,
   onFilesSelection = () => {},
   textExplanation,
+  actionButton: Button,
```

Comment:
I'm not sure. Where is Button declared?

Reply:
It is an aliase.

Reply:
i mean, dk what corresponds to 'Button', but it think the default value might be '<Button/>' and not 'Button'

Reply:
I think I am not getting your point here. It is not a default value it is just a rename of the prop actionButton to be able to use the first letter as a capital letter to detonate it is a component.